examples/use-cases/issue335_pddl/issue335.py

```diff
@@ -22,16 +22,16 @@
     sys.exit(msg)
 
 script_path = tools.get_script_path()
-script_dir = os.path.dirname(script_path)
+script_dir = script_path.parent
```

Comment:
Can we use
```suggestion
script_dir = tools.get_script_dir()
```
and remove the line above if `script_path` is not needed?